code/game/gamemodes/wizard/artefact.dm

```diff
@@ -652,142 +652,174 @@ GLOBAL_LIST_EMPTY(multiverse)
 	item_state = "electronic"
 	origin_tech = "bluespace=4;materials=4"
 	w_class = WEIGHT_CLASS_TINY
-	var/list/spooky_scaries = list()
-	var/unlimited = 0
-	var/heresy = 0
-
-/obj/item/necromantic_stone/unlimited
-	unlimited = 1
+	///List of mobs transformed into skeletons by the stone
+	var/list/active_skeletons = list()
+	///How many skeletons can be converted by the stone at a time
+	var/max_skeletons = 3
+	///If the stone can convert infinite skeletons, bypassing max_skeletons
+	var/unlimited = FALSE
+	///If the stone converts into anime instead of skeletons
+	var/heresy = FALSE
+	///how long the additional_thralls_cooldown is
+	var/above_cap_cooldown = 1 MINUTES
+	///Cooldown between uses when living skeletons above max skeletons
+	COOLDOWN_DECLARE(additional_thralls_cooldown)
+
+/obj/item/necromantic_stone/Destroy()
+	. = ..()
+	active_skeletons = null
 
-/obj/item/necromantic_stone/attack(mob/living/carbon/human/M as mob, mob/living/carbon/human/user as mob)
+/obj/item/necromantic_stone/examine(mob/user)
+	. = ..()
+	var/skele_count = length(active_skeletons)
+	if(skele_count)
+		. += "[skele_count] skeleton thrall[skele_count > 1 ? "s have" : " has"] been risen by [src]."
+	if(unlimited || skele_count < max_skeletons)
+		return
+	var/cooldown_time_left = COOLDOWN_TIMELEFT(src, additional_thralls_cooldown)
+	if(cooldown_time_left)
+		. += "[src] is being strained by the amount of risen skeletons thralls. It cannot be used to rise another skeleton thrall for <b>[DisplayTimeText(cooldown_time_left)]."
```

Comment:
Well you took care of the span which I thank you for. Though now the `<b>` needs to either be removed, or given a closing tag.